modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/facet/AssetEntriesSearchFacet.java

```diff
@@ -217,14 +218,22 @@ public void setServletContext(ServletContext servletContext) {
 			AssetRendererFactoryRegistryUtil.getAssetRendererFactories(
 				companyId);
 
+		String[] engineHelperAllowedEntryClassNames =
+			SearchEngineHelperUtil.getEntryClassNames();
```

Comment:
@istvansajtos We are in a `@Component` so you should be using

```java
@Reference
SearchEngineHelper searchEngineHelper
```

instead of the Util.